tests/bluetooth/shell/src/main.c

```diff
@@ -122,10 +123,20 @@ static void hrs_notify(void)
 
 void main(void)
 {
-	if (IS_ENABLED(CONFIG_USB_UART_CONSOLE)) {
-		usb_enable(NULL);
-		k_sleep(K_SECONDS(2));
+#if DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_shell_uart), zephyr_cdc_acm_uart)
+	const struct device *dev;
```

Comment:
what about moving this to shell backend. There is a `enable` function called from shell thread context. You can block there.
https://github.com/zephyrproject-rtos/zephyr/blob/525fa76f4dd13d42549261c9fc846a150ee291bc/subsys/shell/backends/shell_uart.c#L238

Reply:
However, that will handle shell only and not printk :/ maybe it must stay here or we redirect printk to shell like log is redirected.

Reply:
I would like to keep it that way for now. That is consistent with other examples. Also `CONFIG_SHELL_BACKEND_SERIAL_CHECK_DTR` is enabled, waiting here is general for this sample.